lsp-mode.el

```diff
@@ -480,32 +480,32 @@ the server has requested that."
 ;;;###autoload(put 'lsp-enable-file-watchers 'safe-local-variable #'booleanp)
 
 (defcustom lsp-file-watch-ignored '(; SCM tools
```

Comment:
This change is not related.

Reply:
True. My latest PRs are layered on top of each other, because I needed all of them applied to my local lsp-mode. Do you want me to rebase this one?

Reply:
Yes, please separate them.

Reply:
Okay, I'll do that tomorrow, since I don't have much time right now.